index.html

```diff
@@ -63,5 +63,187 @@ <h1 class="narson">I'm Narson.
     </div>
     </section>
 
+    <!--work section-->
+
+    <section class="work">
+        <div class="card-container">
+            <img class="sary" src="media/Snapshoot-Portfolio-desktop.svg" alt="Project Snapshoot" />
+            
+            <div class="work-fonony">
+
+            <h2 class="card-tittle">Tonic</h2>
+            <div class="frame3-img">
+                <!-- <img src="media/Frame 2.svg" /> -->
+                <ul class="project-info">
+                    <li class="fb">CANOPY</li>
+                    <li class="full">Back End Dev</li>
+                    <li class="year">2015</li>
+                </ul>
+            </div>
```

Comment:
Kindly update your works section to match the figma design better🙏

Your design:
<img width="237" alt="image" src="https://user-images.githubusercontent.com/37116322/235077451-de735549-0f3c-4dda-a569-da1fd04f19be.png">

Figma design:
<img width="249" alt="image" src="https://user-images.githubusercontent.com/37116322/235077547-29a8062f-a52d-49a0-b7d2-576a6e989171.png">

A few things to note:
- Please increase the spacing between the `headline and the works section` and the `works sections and the about section`
- Please increase the spacing between each card
- Please make sure that the project image covers the full width of the card minus the internal card padding